packages/contracts/contracts/newsroom/EventStorage.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.4.19;
+
+contract EventStorage {
+  event StringStored(uint indexed id, string data);
+
+  uint private latestId;
+
+  function store(string data) public {
```

Comment:
I don't think this function should increment latestId. An editor might want to split up the article posting into multiple transactions. Should just emit an event (articleId, revisionId, content)